native/src/@types/react-native-static-server.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'react-native-static-server' {
```

Comment:
there are typings for this package, but they weren't released for a year. This is the copy of their types https://github.com/futurepress/react-native-static-server/blob/master/index.d.ts